docs/operator-manual/upgrading/2.6-2.7.md

```diff
@@ -0,0 +1,37 @@
+# v2.6 to 2.7
+
+## Configure RBAC to account for new `extensions` resource
+
+2.7 introduces the new [Proxy Extensions][1] feature with a new `extensions`
+[RBAC resource][2].
+
+When you upgrade to 2.7, RBAC policies with `*` in the *resource*
+field and `*` in the action field, it will automatically grant the
+`extensions` privilege.
+
+The Proxy Extension feature is disabled by default, however it is
+recommended to check your RBAC configurations to enforce the least
+necessary privileges.
+
+Example
+Old:
+
+```csv
+p, role:org-admin, *, *, my-proj/*, allow
```

Comment:
I think this would be a better example, since extension names can't include slashes:

```csv
p, role:org-admin, *, *, *, allow
```

Reply:
fixed